tests.py

```diff
@@ -2,7 +2,7 @@
 import io
 import unittest
 
-import mock
+from unittest import mock
```

Comment:
there is already a unittest import above and ironically this import is in the wrong order in any case.    since we are py3k only it's fine to use unittest.mock however, just move this import up